molgenis-data/src/test/java/org/molgenis/data/support/DefaultEntityTest.java

```diff
@@ -21,14 +30,72 @@
 	DataService dataService;
 
 	@BeforeMethod
-	public void beforeMethod()
+	public void beforeClass()
```

Comment:
Use BeforeClass annotation and make method static if you want to run this code only once, else rename back to beforeMethod.